src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GlobalExceptionHandler.kt

```diff
@@ -150,6 +152,19 @@ class GlobalExceptionHandler(
     return populateErrorResponseAndHandleExceptionInternal(e, BAD_REQUEST, request)
   }
 
+  /**
+   * Exception handler to return a 503 Service Unavailable ErrorResponse, specifically for a DataAccessException.
+   */
+  @ExceptionHandler(DataAccessException::class)
```

Comment:
New exception handler method to handle any database exception (all `spring-data` exceptions extend `DataAccessException`)